definitions/infra-prometheus-redis/definition.yml

```diff
@@ -0,0 +1,22 @@
+domain: infra
+type: prometheus-redis
```

Comment:
We probably want REDIS here (e.g. this allows us to start synthesizing REDIS's from other telemetry sources, but keep the same type of entities)

Reply:
That makes sense. I changed it to REDIS, let me know if it needs to be something else.